lib/src/screens/exchange_trade/exchange_trade_page.dart

```diff
@@ -378,9 +381,11 @@ class ExchangeTradeState extends State<ExchangeTradeForm> {
                           });
                     },
                     actionLeftButton: () => Navigator.of(context).pop(),
-                    feeFiatAmount: widget.exchangeTradeViewModel.sendViewModel.pendingTransactionFeeFiatAmount
+                    feeFiatAmount: widget.disableFiat ? ''
+                        : widget.exchangeTradeViewModel.sendViewModel.pendingTransactionFeeFiatAmount
                         +  ' ' + widget.exchangeTradeViewModel.sendViewModel.fiat.title,
-                    fiatAmountValue: widget.exchangeTradeViewModel.sendViewModel
+                    fiatAmountValue: widget.disableFiat ? ''
+                        : widget.exchangeTradeViewModel.sendViewModel
```

Comment:
Same as comments for send view model and pendingTransactionFeeFiatAmount, need to add formatted getter which will contains logic for formatted pendingTransactionFeeFiatAmount and logic for case when fiat is disabled.